code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -26,8 +26,12 @@
 /obj/item/soulstone/anybody
 	usability = TRUE
 
+/obj/item/soulstone/anybody/wizard
+	desc = "An artificial replica of a fragment of the legendary treasure known simply as the 'Soul Stone'."
```

Comment:
I feel like "replica" is the wrong word to use here, but I don't know what to replace it with. Suggestions would be appreciated.

Reply:
it's because you used the term artificial before it.

A replica is already a fake by the nature of it being a replica.